src/common/GroupShape.cpp

```diff
@@ -27,11 +33,22 @@ float GroupShape::getRadius() const {
     return mCircle.getRadius();
 }
 
+void GroupShape::setVelocity(sf::Vector2f velocity) {
+    mVelocity = velocity;
+}
+
 void GroupShape::setPosition(sf::Vector2f position) {
     mCircle.setPosition(position);
 }
 
-
 void GroupShape::setRadius(int size) {
     mCircle.setRadius(size);
 }
+
+void GroupShape::move() {
+    mCircle.setPosition(mCircle.getPosition() + mVelocity);
+}
+
```

Comment:
I'd add a comment explaining why we need this method

Reply:
? seems pretty straightforward to me

Reply:
Why is there the one that takes an offset was my q

Reply:
Oh, the offset one is used by the collision handler, this one is used by the group's update method.

Reply:
that's the comment I was thinking